components/aks/aks.tf

```
@@ -82,6 +82,17 @@ module "kubernetes" {
       node_taints         = []
       enable_auto_scaling = true
       mode                = "User"
+    },
+    {
+      name                = "cronjob"
+      vm_size             = "Standard_D4ds_v5"
+      min_count           = 0
+      max_count           = 10
+      max_pods            = 30
+      os_type             = "Linux"
+      node_taints         = ["kubernetes.io/name=cronjob:NoSchedule"]
```

is that name going to be right?

---

I still need at the toleration but plan to use that name and value.

---

I think this may be better:
```suggestion
      node_taints         = ["dedicated=jobs:NoSchedule"]
```

---

didn't realise you don't need the kubernetes.io.  I'll update it now.